src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+:root {
+  --main-font-family: roboto;
+  --main-font-style: medium;
+  --main-font-size: 12px;
+  --Main-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  max-width: 1200px;
```

Comment:
Try to remove max-width, it could help to pass all of the tests

Reply:
when i removed max-width i have 2 tests faild anyway

Reply:
got it